examples/quickstart-scripts/setup-ts-sdk.sh

```
@@ -0,0 +1,243 @@
+  "name": "${PROJECT_NAME}",
+  "version": "1.0.0",
+  "private": true,
+  "type": "module",
+  "scripts": {
+    "dev": "tsx watch src/index.ts",
+    "build": "tsc",
+    "start": "node dist/index.js",
+    "test": "vitest run"
+  },
+  "dependencies": {
+    "`@urbackend/sdk`": "^0.4.2",
```

Shellcheck (0.11.0)

[style] Use $(...) notation instead of legacy backticks `...`.

(SC2006)